configs/default/DAFP-DARWINF4SX1280HD.config

```diff
@@ -0,0 +1,101 @@
+# Betaflight / STM32F411SX1280 (S4SX) 4.3.0 Jun 12 2022 / 14:56:52 (6d17f3f) MSP API: 1.44
+
+board_name DARWINF4SX1280HD
+manufacturer_id DAFP
+
+# resources
+resource BEEPER 1 A14
+resource MOTOR 1 B07
+resource MOTOR 2 B10
+resource MOTOR 3 A00
+resource MOTOR 4 B08
+resource MOTOR 5 A08
+resource MOTOR 6 A10
+resource LED_STRIP 1 B01
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 11 A08
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 11 A08
+resource LED 1 C14
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource ADC_BATT 1 A01
+resource ADC_CURR 1 B00
+resource OSD_CS 1 B12
+resource RX_SPI_CS 1 A15
+resource RX_SPI_EXTI 1 C13
+resource RX_SPI_BIND 1 B02
+resource RX_SPI_LED 1 C15
+resource RX_SPI_EXPRESSLRS_RESET 1 B09
+resource RX_SPI_EXPRESSLRS_BUSY 1 A13
+resource GYRO_EXTI 1 B06
+resource GYRO_CS 1 A04
+
+# timer
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer A00 AF1
+# pin A00: TIM2 CH1 (AF1)
+timer B10 AF1
+# pin B10: TIM2 CH3 (AF1)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer A10 AF1
+# pin A10: TIM1 CH3 (AF1)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin A00 0
+# pin A00: DMA1 Stream 5 Channel 3
+dma pin B10 0
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+
+# feature
+feature SOFTSERIAL
+feature RX_SPI
+feature TELEMETRY
+feature OSD
+
+# master
+set rx_spi_bus = 3
+set rx_spi_led_inversion = ON
+set dshot_burst = AUTO
+set dshot_bitbang = OFF
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 125
+set beeper_inversion = ON
+set beeper_od = OFF
+set small_angle = 60
+set system_hse_mhz = 8
+set max7456_spi_bus = 2
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW270
+set gyro_1_align_yaw = 2700
+set expresslrs_domain = ISM2400
+set name = DarwinFPV

```

Comment:
What's this?

Reply:
```
configs/default/NEBD-NBD_INFINITYF4.config:118:set name = INFINITY
configs/default/TTRH-TRANSTECF405HD.config:108:set name = TransTEC
configs/default/TTRH-TRANSTECF411AIO.config:122:set name = TransTEC
configs/default/TTRH-TRANSTECF411HD.config:110:set name = TransTEC
configs/default/AIRB-HELSEL_STRIKERF7.config:91:set name = StrikerF7
configs/default/RCTI-ARESF7.config:109:set name = Ares F7
configs/default/TTRH-TRANSTECF7HD.config:109:set name = TransTEC

```

Reply:
Cleanup: https://github.com/betaflight/unified-targets/pull/636